index.html

```diff
@@ -16,6 +16,18 @@ <h1>Rev My $</h1>
     </div>
     
     <h3>Hello, yourName</h3>
+
+     <!--monthly income input  -->
+     <div class="goalsAndExpenses">
+        <label for="MonthlyIncome">Monthly Income</label>
+        <input class="form-control " name="income" id="monthlyIncome" value="" autofocus placeholder="Monthly Income">
+    </div>
+
+    <!--monthly savings goal input  -->
+    <div class="monthlySavings">
```

Comment:
if no .monthlySavings class in the css please delete for now

Reply:
The class has been created in HTML to be referenced in CSS later on.